src/Renderer/ThreeExtended/OBB.js

```diff
@@ -98,6 +98,18 @@ OBB.prototype._cPointsWorld = function _cPointsWorld(points) {
     return points;
 };
 
+OBB.prototype.sphereIsAboveXYBox = function sphereIsAboveXYBox(sphere) {
```

Comment:
Functions name start with a verb (`isXXX`).

Can you add the jsdoc too please?

Reply:
done